src/universal/components/Button/Button.js

```diff
@@ -198,6 +201,10 @@ class Button extends Component {
   }
 }
 
+const icon = {
+  fontSize: ui.iconSize2x
```

Comment:
I think we should still use line 160 `fontSize: ui.buttonIconSize[size]` without reworking any further to handle icon size. And it really should be `fontSize: ui.buttonIconSize[size] || ui.buttonIconSize.medium` where `medium` is the fallback if size is not specified. I’m trying to remember if the `defaultProps` object works with our current styling pattern. Perhaps it doesn’t work in the style thunk.

Reply:
Ah I see. That sounds perfect to me. I think that I made this change because, for whatever odd reason, the icon's fontSize ended up being 0. I'll look into this further on the new PR.

Reply:
Ah I see. This was because I was not passing in the `size` prop.

Reply:
Yeah, and I dropped the ball, should’ve declared that default `ui.buttonIconSize.medium` when no size prop has been passed in.